src/main/groovy/com/inxmail/clicktracker/controller/ApiController.groovy

```diff
@@ -0,0 +1,39 @@
+package com.inxmail.clicktracker.controller
+
+import com.inxmail.clicktracker.entity.PostUrl
+import com.inxmail.clicktracker.entity.Url
```

Comment:
Yes you are right. We could use Url entity also for POST, dont know what I thought.